server/api.js

```diff
@@ -10,6 +10,7 @@ const allowlist = {
 	POST: {
 		"/sign-in": "public",
 		"/tender": "buyer",
+		"/bid": "token",
```

Comment:
Since a request for this endpoint looks like `/bid/123/status` and we want to match only `bid`, we need to split the request path and grab only the first part before matching against our allow list